dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -727,6 +729,7 @@ public Workflow publish(@ApiParam(hidden = true) @Auth User user,
             c.setIsPublished(false);
             if (checker != null) {
                 checker.setIsPublished(false);
+                elasticManager.handleIndexUpdate(checker, ElasticMode.DELETE);
```

Comment:
Whether the checker workflow has been changed from unpublished to published or vice versa should be irrelevant.  It should not make it into the index at all.